README.md

```diff
@@ -187,7 +187,7 @@ Contributing
 Here is the Clickhouse schema to run the indexer:
 
 ```sql
-CREATE TABLE defuse_assets (
+CREATE TABLE defuse_indexer.defuse_assets (
```

Comment:
Please revert the changes done to README around the schemas definition. If you use the `USE <database_name>;` You don't need to provide a full name for these queries. This change is redundant.